Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+
+# Install Python and necessary dependencies
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip git && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
+    pip3 install setuptools
+
+ENV PYTHONPATH=/scrutinycspm:/scrutinycspm/src:/scrutinycspm/cli
+
+WORKDIR /scrutinycspm
+
+COPY setup.py .
+COPY . .
+
+RUN pip3 install --no-cache-dir -e .
+EXPOSE 5678
```

Comment:
We probably don't need to expose a port at the moment. You should be able to you shell into the container to use the CLI. We would only expose a port if using a web connection

Reply:
Expose 5678 is a leftover from trying to configure according to Microsoft VS code instructions.  I will remove it.